cycler.py

```diff
@@ -335,7 +335,11 @@ def __repr__(self):
     def _repr_html_(self):
         # an table showing the value of each key through a full cycle
         output = "<table>"
-        sorted_keys = sorted(self.keys)
+        try:
+            sorted_keys = sorted(self.keys)
```

Comment:
This approach would also help in situations where cyclers have mixed types for keys because in py3k, they would not be orderable as-is.